doc/src/selections.rst

```diff
@@ -72,6 +79,11 @@ Additional selectors ideas are welcome if they cannot be created easily from a
 combinaison of the above. If you don't know if your selector idea can be
 created from a combinaison, just ask!
 
+Supported functions in mathematical expressions are: ``sin``, ``cos``, ``tan``
+for the trigonometric functions; ``asin`` and ``acos`` reciprocal trigonometric
```

Comment:
I'm confused. By reciprocal do you mean CSC, SEC, and COT? You mention the Arc trig functions by name, so I'm a bit lost.

Reply:
You also don't mention atan. This function is a bit tricky as there are two implementations of it in C++ ( tan() and tan2()) that behave differently. It may be good to mention which one you've used specifically.

Reply:
> By reciprocal do you mean CSC, SEC, and COT? You mention the Arc trig functions by name, so I'm a bit lost.

For me, reciprocal means the one going from the sin value to the angle. How would you call them?

> You also don't mention atan. This function is a bit tricky as there are two implementations of it in C++ ( tan() and tan2()) that behave differently. It may be good to mention which one you've used specifically.

Yeah, `atan` is tricky, so I left it out 😃

Reply:
To me a reciprocal trig function is defined like so:

`sec(a) = 1 / cos(a)` or `csc(a) = 1/csc(a)`. See (https://www.khanacademy.org/math/geometry-home/right-triangles-topic/reciprocal-trig-ratios-geo/a/reciprocal-trig-ratios) for details (Khan academy can explain it better than I can).

What you describe is an inverse trig function (sometimes called an Arcus trig function, hence the 'a' in front of the name). They are defined as the graph of `x = sin(y)`, or `y=arcsin(x)` - whichever you prefer. See (https://en.wikipedia.org/wiki/Inverse_trigonometric_functions) for details.